### ISSUE TITLE
Minor clarification for aria-expanded

### ISSUE THREAD

**Issue by scottaohara** [w3c/aria#1912]
concerning the following text:
>If a grouping container that can be expanded or collapsed is not [owned](https://www.w3.org/TR/wai-aria-1.2/#dfn-owned-element) by the element that has the [aria-expanded](https://www.w3.org/TR/wai-aria-1.2/#aria-expanded) attribute, the author SHOULD identify the controlling relationship by referencing the container from the element that has [aria-expanded](https://www.w3.org/TR/wai-aria-1.2/#aria-expanded) with the [aria-controls](https://www.w3.org/TR/wai-aria-1.2/#aria-controls) property.

The use of the 'owned' term seems to imply that a container element could expose an expanded/collapsed state, but this doesn't really jive with most of the roles that allow the aria-expanded state - save for treeitem which already goes over this parent child relationship in the paragraph previous to the quoted.  Rather, this paragraph could be interpreted (spoiler, it has) as saying that one could do either:

```
<button aria-expanded=...>
   button name
   <div>
     content to expand or collapse
   </div>
</button>
```

or

```
<button aria-expanded=... aria-owns=f>
   button name
</button>
<div id=f>
   content to expand or collapse
</div>
```

if not using aria-controls.

It'd be nice if we could revise that paragraph a bit so as to mitigate others thinking the above examples would be acceptable.

**Comment by adampage** [comment]
@scottaohara, do you think it’d be too prescriptive (or just wrong?) to _**require**_ `aria-controls` for the subset of supported roles where “ownership” isn’t applicable?

Something like so?:

> When `aria-expanded` is used on an element of any of the following roles,
> authors _**MUST**_ additionally include `aria-controls` to identify the
> specific grouping element which will be expanded or collapsed:
> 
> - button
> - checkbox
> - combobox
> - link
> - listbox
> - menuitem
> - menuitemcheckbox
> - menuitemradio
> - tab
> - switch

**Comment by scottaohara** [comment]
i do not think it is a _must_, no.  particularly since in many cases there is no overt user benefit to whether `aria-controls` has been used or not.

**Comment by adampage** [comment]
Thanks, @scottaohara. Here’s a second attempt:

> ~~If a grouping container that can be expanded or collapsed is not [owned](https://www.w3.org/TR/wai-aria-1.2/#dfn-owned-element) by the element that has the [aria-expanded](https://www.w3.org/TR/wai-aria-1.2/#aria-expanded) attribute, the author SHOULD identify the controlling relationship by referencing the container from the element that has [aria-expanded](https://www.w3.org/TR/wai-aria-1.2/#aria-expanded) with the [aria-controls](https://www.w3.org/TR/wai-aria-1.2/#aria-controls) property.~~

> Authors SHOULD set `aria-controls` on the same interactive element as `aria-expanded`, thereby identifying which content container it will expand and collapse. Authors MAY omit `aria-controls` for specific roles where the controlling relationship is implicit, such as `row` and `treeitem`.

It’s still a bit handwavy, but feels less permissive toward the bad examples you identified. I almost re-incorporated the concept of “owning” into the second sentence (to clarify how a controlling relationship could be implicit) but decided that muddied things unnecessarily.

Anyhoo, I’ll get a PR going on this to open it up for wordsmithing.
